### ISSUE TITLE
derivative of the matrix in FeaturePoint6d(Relative)::computeErrorDot

### ISSUE THREAD

**Issue by francois-keith** [stack-of-tasks/sot-core#10]
Hello everyone

I have a doubt on the computation of the vector errordot for the classes FeaturePoint6d and FeaturePoint6dRelative.
In particular, I'd like to have some explanations on the computation of the 3 last values of this vector, corresponding to the angular velocity.

For the class FeaturePoint6d, this vector is computed as follows:

<pre>
wdRp is the rotation part of the homogeneous matrix corresponding to the derivative (an input).
dRerr = transpose (wdRp)
rerr.fromMatrix( dRerr )
</pre>

The problem is: how should be computed the matrix wdRp ?
Typically, if we use the formula `dR = skew(w).R` (The one used in sot-pattern-generator for example), there is an issue:

if the angular velocity is null (w=0), the derivative of the rotation matrix dR is also null.
This creates an issue for the computation of rerr, since dRerr does not correspond to a correct rotation matrix (its determinant equals 0). As a result, rerr has a unexpected value (1.8138,1.8138,1.8138)

The same issue can be seen in FeaturePoint6dRelative

To sum up the problem:
The error "rerr" cannot be associated to the angular velocity w, when dR has the form dR=skew(w)*R 
Does rerr correspond to the angular velocity?
If so, is that an error in the code? 
If not, what is the definition of the derivative of the rotation dR? In which scripts this feature has already been used?

Thanks

**Comment by jmirabel** [comment]
FeaturePoint6d is now deprecated. FeaturePose should be used instead.
